consultation_analyser/consultations/api/serializers.py

```diff
@@ -0,0 +1,60 @@
+from rest_framework import serializers
+
+
+class DemographicOptionsSerializer(serializers.Serializer):
+    demographic_options = serializers.DictField(
```

Comment:
this is bit of an anti pattern having arbitrarily named keys in a payload but is a consequence of the model. could we consider a payload shaped more like the model itself:

```json
[
    {"consultation_id": ..., "name": "gender", "value": "male"},
    {"consultation_id": ..., "name": "gender", "value": "female"}
]
```